sklearn/metrics/_regression.py

```diff
@@ -1258,7 +1313,7 @@ def max_error(y_true, y_pred):
     np.int64(1)
     """
     xp, _ = get_namespace(y_true, y_pred)
-    y_type, y_true, y_pred, _ = _check_reg_targets(y_true, y_pred, None, xp=xp)
+    y_type, y_true, y_pred, _ = _check_reg_targets(y_true, y_pred, None)
```

Comment:
why not pass the xp here?

Reply:
It was my mistake. Thanks for catching that!